### PR TITLE
Add a Hello World application that can be used and the initial Linux process

### PR DESCRIPTION
**Pull request by conradgrobler**
The initial Linux process  (PID 1) must perform some basic environment initialization to ensure the rest of the application runs as expected. This is the first step towards building and testing a support library for doing that initialization.

### CODE REVIEW

File: testing/hello_world_linux_init/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[package]
```
Comment (tiziano88) [inline testing/hello_world_linux_init/Cargo.toml:1]: Personally I would move this crate up to the top level and call it `oak_hello_world_linux_init` or similar.

File: Cargo.toml [hunk 2]
```diff
@@ -50,6 +50,7 @@ members = [
   "oak_virtio",
   "sev_serial",
   "snp_measurement",
+  "testing/hello_world_linux_init",
```
Comment (ipetr0v) [inline Cargo.toml:53]: Will this hello world always be the initial process from which everything is forked?
Or this one is just for testing/experimenting?

File: testing/hello_world_linux_init/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<!-- Oak Logo Start -->
+<!-- An HTML element is intentionally used since GitHub recommends this approach to handle different images in dark/light modes. Ref: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to -->
+<!-- markdownlint-disable-next-line MD033 -->
+<h1><picture><source media="(prefers-color-scheme: dark)" srcset="docs/oak-logo/svgs/oak-logo-negative.svg?sanitize=true"><source media="(prefers-color-scheme: light)" srcset="docs/oak-logo/svgs/oak-logo.svg?sanitize=true"><img alt="Project Oak Logo" src="docs/oak-logo/svgs/oak-logo.svg?sanitize=true"></picture></h1>
+<!-- Oak Logo End -->
+
+# Initial Process "Hello World" Application
```
Comment (ipetr0v) [inline testing/hello_world_linux_init/README.md:7]: If it's the initial process, should it be called `init` or `zygote` ?
Does it actually print out hello world?

**Review by ipetr0v — APPROVED** [review]

File: testing/hello_world_linux_init/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+## Testing Steps
+
+Note: these steps assume that the commands will be executed from the project
+root.
+
+Build the application as a statically linked binary:
+
+```bash
+
```
Comment (tiziano88) [inline testing/hello_world_linux_init/README.md:34]: there should not be extra lines inside the code block fences (only outside)

**Review by tiziano88 — APPROVED** [review]

File: Cargo.toml (hunk 2 above)
Comment (conradgrobler) [reply Cargo.toml:53]: This is just for initial testing and experimenting.

File: testing/hello_world_linux_init/README.md (hunk 3 above)
Comment (conradgrobler) [reply testing/hello_world_linux_init/README.md:7]: This is just a test application that logs "Hello world." to the console.

**Review by conradgrobler — COMMENTED** [review]
Thanks.

File: testing/hello_world_linux_init/Cargo.toml (hunk 1 above)
Comment (conradgrobler) [reply testing/hello_world_linux_init/Cargo.toml]: Done.

**Pull request merged**